dxf2png/main.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright (c) 2021 Nikita Letov (letovnn@gmail.com)
+Copyright (c) 2021 GRAD4 Inc.

```

Comment:
@jalovisko please keep this under your name. You've done the work and therefore the MIT copyright should be under your name.

Reply:
ok, got it!